packages/neon_framework/packages/talk_app/lib/src/widgets/rich_object/file.dart

```diff
@@ -35,8 +32,8 @@ class TalkRichObjectFile extends StatelessWidget {
 
         if (parameter.width != null && parameter.height != null) {
           deviceSize = Size(
-            _parseDimension(parameter.width!).toDouble(),
-            _parseDimension(parameter.height!).toDouble(),
+            int.parse(parameter.width!).toDouble(),
+            int.parse(parameter.height!).toDouble(),
```

Comment:
```suggestion
            double.parse(parameter.width!),
            double.parse(parameter.height!),
```

Reply:
I did this to make sure the values are valid integers.
But yeah we shouldn't do this kind of enforcing that only tests the API and not our code.